svthalia/concrexit
# Manage npm based packages with yarn

Closes #1610 

### Summary
Updates jquery and makes sure it stays updated via yarn. For the latter part renovate is enabled for javascript.

### How to test
Steps to test the changes you made:
1. Go to any page
2. See no javascript/css loading errors
